app/plugins/prometheus.md

```diff
@@ -0,0 +1,171 @@
+---
+id: page-plugin
+title: Plugins - Prometheus
+header_title: Prometheus
+header_icon: https://konghq.com/wp-content/uploads/2018/06/prometheus.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Usage
+    items:
+      - label: Log Format
+description: |
+  This plugin exposes metrics in [Prometheus Exposition format](https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md). It allows you to integrate with your existing monitoring and alerting system.
+
+params:
+  name: prometheus
+
+---
+
+
+#### Enable the plugin
+
+```bash
+$ curl http://localhost:8001/plugins name=prometheus
+```
+
+#### Reading metrics into Prometheus
```

Comment:
Can we add the bit about the metrics which are available in this plugin right now?
For reference, we can use https://github.com/kong/kong-plugin-prometheus#available-metrics

Reply:
Done